.github/workflows/certora.yml

```diff
@@ -13,20 +13,35 @@ on:
   workflow_dispatch:
 
 jobs:
-  verify_stkBNB:
+  get-diff:
     runs-on: ubuntu-latest
-
+    # Map a step output to a job output
+    outputs:
+      output1: ${{ steps.certora.outputs.run }}
+    
     steps:
       - uses: actions/checkout@v3
 
       - name: Get Diff Action
         uses: technote-space/get-diff-action@v4.0.1
-        id: git-diff # via outputs
+        id: get-diff-action # via outputs
         with:
           PATTERNS: |
             +(contracts)/*.sol
             +(contracts|certora)/**/*.sol
 
+      - id: certora
+        run: echo "::set-output name=run::true"
+        if: steps.get-diff-action.outputs.diff
```

Comment:
I believe, these extra steps can be avoided. Just the get-diff step should be sufficient.

Instead of creating and exporting `steps.certora.outputs.run` in the job `outputs`, you can directly export `steps.get-diff-action.outputs.diff`

Reply:
done